workshops/forms.py

```diff
@@ -412,3 +413,18 @@ class Meta:
         model = Task
         fields = '__all__'
         widgets = {'person': HiddenInput}
+
+
+class HostForm(forms.ModelForm):
+    domain = forms.CharField(
+        max_length=Host._meta.get_field('domain').max_length,
+        validators=[
+            RegexValidator('[^\w\.-]+', inverse_match=True,
```

Comment:
`RegexValidator` throws `ValidationError` when it doesn't match and `inverse_match == False`.
It throws `ValidationError` when it does match and `inverse_match == True`.

Now, I want the error to show up when any of these chars is encountered: `:/` (and others, but let's keep the example short).

So if I used `[\w]+` and `inverse_match == False` then:
* `http` would pass
* `http://` would pass
* `:/` wouldn't pass

I only want to show errors if the wrong characters are found in the `Host.domain`. If I used `[\w]+` instead of `[^\w]+` it would mean "show error when *good* chars are in the Host.domain".